features/car/service.feature

```diff
@@ -4,11 +4,11 @@ Feature: Car - Surfaces
     Background:
         Given the profile "car"
 
-    Scenario: Car - Surface should reduce speed
+    Scenario: Car - Ways tagged service should reduce speed
         Then routability should be
-            | highway  | service           | forw       | backw       |
-            | service  | alley             | 5 km/h +-1 | 5 km/h +-1  |
-            | service  | emergency_access  |            |             |
-            | service  | driveway          | 5 km/h +-1 | 5 km/h +-1  |
-            | service  | drive-through     | 5 km/h +-1 | 5 km/h +-1  |
-            | service  | parking           | 5 km/h +-1 | 5 km/h +-1  |
+            | highway  | service           | forw        | backw        | forw_rate  |
+            | service  | alley             | 15 km/h +-1 | 15 km/h +-1  | 2          |
+            | service  | emergency_access  |             |              |            |
```

Comment:
Now that we have the ability to penalize access-limited ways, should `emergency_access` just become like all the others (i.e. allow routing if a pin is dropped) ?

On second thought, I guess not - cut-off roads on freeways that are emergency-only should never be routed over, and because they're short, even heavily penalizing them is not sufficient to prevent access.  We would need to implement turn-based penalties for that to work.

So, nevermind, but I've written all this stuff now, so I'm going to post it anyway for perpetuity.